src/main/java/hudson/scm/SubversionSCM.java

```diff
@@ -463,6 +463,13 @@ public void setWorkspaceUpdater(WorkspaceUpdater workspaceUpdater) {
     public ModuleLocation[] getLocations(AbstractBuild<?,?> build) {
         return getLocations(null, build);
     }
+    
+    /**
+     * Fix for the backward-compatibility with svn-tag plugin
```

Comment:
"Fix for..." is a good remark, but not a summary
add <code>@since</code>. If the method existed before, it worth to mention it in remarks